src/frontend/apps/desk/src/components/BoxButton.tsx

```diff
@@ -18,7 +18,7 @@ export type BoxButtonType = ComponentPropsWithRef<typeof BoxButton>;
  * ```
  */
 const BoxButton = forwardRef<HTMLDivElement, BoxType>(
-  ({ $css, ...props }, ref) => {
+  ({ $css, tabIndex = 0, ...props }, ref) => {
```

Comment:
I don't think you need it, `tabIndex` is already a prop of `BoxButton`. 
```suggestion
  ({ $css, ...props }, ref) => {
```